src/pages/workspaces/migration/BillingProjectParent.ts

```diff
@@ -14,17 +14,17 @@ import {
   getBillingProjectMigrationStats,
   inProgressIcon,
   successIcon,
+  WorkspaceWithNamespace,
 } from 'src/pages/workspaces/migration/migration-utils';
 import { WorkspaceItem } from 'src/pages/workspaces/migration/WorkspaceItem';
 import { useMemo } from 'use-memo-one';
 
 interface BillingProjectParentProps {
   billingProjectMigrationInfo: BillingProjectMigrationInfo;
+  migrationStartedCallback: (p: WorkspaceWithNamespace[]) => {};
 }
 
 export const BillingProjectParent = (props: BillingProjectParentProps): ReactNode => {
-  const [migrateStarted, setMigrateStarted] = useState<boolean>(false);
```

Comment:
I removed the disabling of the billing project "migrate all…" button because there is not a good way to know when to re-enable it after refreshing the status of in-progress workspaces.

I have kept it for individual workspaces since we know when their state changes, but since we also refresh immediately after a migrate button is pressed, you really don't see the disabled state there anymore.